thrust/thrust/system/cuda/detail/async/inclusive_scan.h

```diff
@@ -126,6 +126,93 @@ async_inclusive_scan_n(execution_policy<DerivedPolicy>& policy, ForwardIt first,
   return ev;
 }
 
+template <typename DerivedPolicy,
+          typename ForwardIt,
+          typename Size,
+          typename OutputIt,
+          typename InitialValueType,
+          typename BinaryOp>
+unique_eager_event async_inclusive_scan_n(
+  execution_policy<DerivedPolicy>& policy, ForwardIt first, Size n, OutputIt out, InitialValueType init, BinaryOp op)
+{
+  using InputValueT = cub::detail::InputValue<InitialValueType>;
+  using AccumT      = typename ::cuda::std::
+    __accumulator_t<BinaryOp, typename ::cuda::std::iterator_traits<ForwardIt>::value_type, InitialValueType>;
```

Comment:
Important: It's funny that I mentioned that i would have swapped the parameters of `__accumulator_t` and you actually supply them in the wrong (IMO right) order :D

So please either call this:
```suggestion
  using AccumT      = typename ::cuda::std::
    __accumulator_t<BinaryOp, InitialValueType, typename ::cuda::std::iterator_traits<ForwardIt>::value_type>;
```
Or swap the parameters of `__accumulator_t`.